cmd/testpeers.go

```diff
@@ -570,12 +570,21 @@ func peerPingLoadTest(ctx context.Context, conf *testPeersConfig, tcpNode host.H
 	return testRes
 }
 
-func dialTCPIP(ctx context.Context, address string) error {
+func dialLibp2pTCPIP(ctx context.Context, address string) error {
 	d := net.Dialer{Timeout: time.Second}
 	conn, err := d.DialContext(ctx, "tcp", address)
 	if err != nil {
 		return errors.Wrap(err, "net dial")
 	}
+	buf := new(strings.Builder)
```

Comment:
Hmm a malicious server could keep the connection open forever. I think we should read a fixed amount of bytes, and check if the string begins with `/multistream/1.0.0`. WDYT?

Reply:
Hmmm yap, that can happen. The server will always be our own though, so it shouldn't be an issue here. I will change it though, better be more agnostic.